crates/node_binding/binding.d.ts

```diff
@@ -504,7 +504,7 @@ export interface RawEntryItem {
 
 export interface RawExperiments {
   lazyCompilation: boolean
-  incrementalRebuild: boolean
+  incrementalRebuild: RawIncrementalRebuild
```

Comment:
It seems to be a breaking change, could you make this field a union? You can leverage `napi::Either`

Reply:
Didn't see you have changed it on the JS side. Just ignore me